### PR TITLE
update to 1.4.10

### PR DESCRIPTION
**Pull request by i-walker**
- Ignores Gradle Repo test, which is targeting 1.4.0 not 1.4.10

### CODE REVIEW

File: idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/repo/GradleTypeproofsTest.kt [hunk 1]
```diff
@@ -4,10 +4,11 @@ import arrow.meta.ide.gradle.GradleTestSetUp
 import arrow.meta.ide.testing.IdeEnvironment
 import com.intellij.openapi.module.ModuleManager
 import org.assertj.core.api.Assertions.assertThatCode
+import org.junit.Ignore
 import org.junit.Test
 
 class GradleTypeproofsTest : GradleTestSetUp() {
-
+  @Ignore
```
Comment (rachelcarmena) [inline idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/repo/GradleTypeproofsTest.kt:11]: Thanks @i-walker :clap: Please, take a look at this code in this test:

```
      // TODO: remove after merging https://github.com/arrow-kt/arrow-typeproofs/pull/10
      val checkoutResult = gitCheckout(myProject, "1.4.0-rc", myProjectRoot)
```

This test is using the branch `1.4.0-rc` in `arrow-typeproofs` repository so you could do the same. I mean, adapt the `arrow-typeproofs` repository for 1.4.10 in a branch and use it here.

Thank you so much!!

**Comment by rachelcarmena** [comment]
>     * Ignores Gradle Repo test, which is targeting 1.4.0 not 1.4.10

It's targeting 1.4.0 in `1.4.0-rc` branch. I added a comment with the details :raised_hands: Thanks @i-walker !!

File: idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/repo/GradleTypeproofsTest.kt (hunk 1 above)
Comment (i-walker) [reply idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/repo/GradleTypeproofsTest.kt]: :D Thanks

**Review by rachelcarmena — APPROVED** [review]
:tada:

**Pull request merged**